IdentityCore/src/logger/MSIDLogger.m

```diff
@@ -204,7 +204,7 @@ - (void)logWithLevel:(MSIDLogLevel)level
             
             if (self.nsLoggingEnabled)
             {
-                NSString *logLevelStr = [self stringForLogLevel:_level];
+                NSString *logLevelStr = [self stringForLogLevel:self.level];
```

Comment:
will this have any additional impact? accessor for the "level" would go through logger connector. Not sure what is the correct design here. @antrix1989 FYI.

Reply:
Based on our discussion with Serhii during the scrum on 8/28/2020, it is safe to keep it as self.level and trigger getter api, as there's no harm in calling it.